demo/screens/EthereumQuestion.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Image, Button, StyleSheet, View, Text } from "react-native";
+import { createStackNavigator, createAppContainer } from "react-navigation";
+
+export default class EthereumQuestion extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.questionText}>{`Are you new to Ethereum?`}</Text>
+        <View style={styles.buttonView}>
```

Comment:
This could be a component called MultiButtonHolder or something. But maybe that's excessive componentization

Reply:
I'll keep that way for now, ok?

Reply:
Yep. Thanks.